osgiify/pom.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons</groupId>
+    <artifactId>org.openhab.addons.reactor</artifactId>
+    <version>2.5.0-SNAPSHOT</version>
+    <relativePath>../poms/bnd</relativePath>
+  </parent>
+
+  <groupId>org.openhab.addons.osgiify</groupId>
+  <artifactId>org.openhab.addons.reactor.osgiify</artifactId>
+  <packaging>pom</packaging>
+
+  <name>openHAB Add-ons :: OSGi-ify</name>
+
+  <modules>
+    <module>com.github.calimero.calimero-core</module>
+  </modules>
+
+  <properties>
+    <osgiify.import></osgiify.import>
+    <osgiify.export></osgiify.export>
+  </properties>
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-jar-plugin</artifactId>
+          <configuration>
+            <archive>
+              <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
+            </archive>
+            <skipIfEmpty>true</skipIfEmpty>
+          </configuration>
+        </plugin>
+        <plugin>
+          <groupId>org.openhab.tools.sat</groupId>
+          <artifactId>sat-plugin</artifactId>
+          <version>${sat.version}</version>
+          <executions>
+            <execution>
+              <id>sat-all</id>
+              <phase>none</phase>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <groupId>com.googlecode.addjars-maven-plugin</groupId>
+          <artifactId>addjars-maven-plugin</artifactId>
+          <version>1.0.5</version>
+          <executions>
+            <execution>
+			  <id>add-libs</id>
```

Comment:
Strange tabs here 😄

Reply:
There are a lot of wrong indentation in that file.